test/api-test/api/collections/post.json

```diff
@@ -0,0 +1,345 @@
+[
+  {
+      "type": "RichSkillCollection",
+      "name": "API Test Collection - System Admin",
+      "id": "-http://localhost:8080/api/collections/9b367b9a-6b27-488c-89b9-3e7228a16960",
```

Comment:
These hyphens are a concern. They should actually be resolved locally on your feature branch via changes in application-dev.properties.